### PR TITLE
Feature/alerts screen refactor

### PR DESCRIPTION
**Pull request by rinslow**
Major work done in this commit:
* Created a new Threshold configuration screen, (now there is a single button that opens up the new screen)
* Refactored Threshold object (see below)

Minor works:
* Added "off" option to all thresholds, meaning they won't be checked
* Made json file readable
* Restructured json file (added volume and bpm, and also every threshold defines it's own step size)
* Graph's Y Scaling moved to Graph Class
* DataStore doesn't read config in `__init__()`, but goes with default values, to read from config it uses a static constructor called load_from_config, this is made so the program won't fail because of a .json file error. The error is logged either way.
* Threshold object now has `min`, `max`, `step` and an `off` value
* Added Breath Volume and BPM Thresholds
* Renamed `AirOutputSummary` to `VolumeSummary`

Included:
screenshots of new program
Screenshot from 2020-03-25 22-47-06

Screenshot from 2020-03-25 22-47-10

New JSON File:
```json
{
  "threshold": {
    "flow": {
      "min": -0.5,
      "max": 14.5,
      "step": 0.5
    },
    "volume": {
      "min": 0.3,
      "max": 26.5,
      "step": 0.5
    },
    "pressure": {
      "min": 0.3,
      "max": 26.5,
      "step": 0.5
    },
    "bpm": {
      "min": 0.3,
      "max": 26.5,
      "step": 0.5
    },
    "breathing_threshold": 3.5
  },
  "log_enabled": true,
  "graph_seconds": 12.0
}

```

### CODE REVIEW

File: graphics/alert_bar.py [hunk 1]
```diff
@@ -15,8 +15,8 @@ class IndicatorAlertBar(object):
     error_dict = {
         alerts.AlertCodes.PRESSURE_LOW: "LOW PIP",
         alerts.AlertCodes.PRESSURE_HIGH: "HIGH PIP",
-        alerts.AlertCodes.BREATHING_VOLUME_LOW: "LOW Ve",
-        alerts.AlertCodes.BREATHING_VOLUME_HIGH: "HIGH Ve"
+        alerts.AlertCodes.BREATHING_VOLUME_LOW: "LOW MVI",
+        alerts.AlertCodes.BREATHING_VOLUME_HIGH: "HIGH MVI"
```
Comment (avrahamshukron) [inline graphics/alert_bar.py]: We are going to have a conflict here. I renamed them to plain English (i.e High Volume, High Pressure).
We should consult with the team doctor.

File: main.py [hunk 2]
```diff
@@ -16,10 +16,10 @@ def configure_logging(level, store):
     logger.setLevel(level)
     # create file handler which logs even debug messages
     fh = RotatingFileHandler('inhalator.log', maxBytes=1024 * 100, backupCount=3)
-    fh.setLevel(logging.ERROR)
+    fh.setLevel(logging.INFO)
```
Comment (avrahamshukron) [inline main.py]: I changed that to DEBUG (as per the comment above) in #12

File: main.py [hunk 3]
```diff
@@ -16,10 +16,10 @@ def configure_logging(level, store):
...
     # create console handler with a higher log level
     ch = logging.StreamHandler()
-    ch.setLevel(logging.ERROR)
+    ch.setLevel(logging.INFO)
```
Comment (avrahamshukron) [inline main.py]: I changed that to `level` in #12 so it would be controllable from command line

File: main.py (hunk 2 above)
Comment (rinslow) [reply main.py]: Used yours then

File: main.py (hunk 3 above)
Comment (rinslow) [reply main.py]: used yours

**Pull request merged**